cli/cmd/migrate_v0.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	source "github.com/cloudquery/plugin-sdk/clients/source/v0"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/rs/zerolog/log"
+)
+
+func migrateConnectionV0(ctx context.Context, cqDir string, sourceSpec specs.Source, destinationsSpecs []specs.Destination) error {
```

Comment:
I would avoid this for now I think it will cause all things like cyclical imports but also on the CLI I don't think it should be different packages as we don't import it so it should only be on the SDK.